### PR TITLE
Restore protected routes

### PR DESCRIPTION
**Pull request by SimonjShurety**

### CODE REVIEW

**Review by koral14 — APPROVED** [review]
Great job, Simon! Thank you!

File: src/App.js [hunk 1]
```diff
@@ -63,25 +60,76 @@ const App = () => {
...
+                                />
+                                <Route
+                                    path="/my-profile"
+                                    element={
+                                        <ProtectedRoute requiredAuthLevel="user">
+                                            <ProfilePage />
+                                        </ProtectedRoute>
+                                    }
+                                />
                             </Routes>
```
Comment (Ekaterina-Bondareva) [inline src/App.js:133]: Could you please add Protected route for chat routes that were accidentally removed? Also could you please wrap HomePage with Protected route?

**Review by Ekaterina-Bondareva — CHANGES_REQUESTED** [review]
Could you please add Protected route for chat routes that were accidentally removed? Also could you please wrap HomePage with Protected route?

File: src/App.js (hunk 1 above)
Comment (SimonjShurety) [reply src/App.js:133]: We had made a conscious decision to leave HomePage unprotected. Let's double check with Jacob/Nicki regarding that change.

**Review by Ekaterina-Bondareva — APPROVED** [review]
Great job, Simon! Thank you!

**Pull request merged**